Cargo.toml

```diff
@@ -8,7 +8,7 @@ version = "2.7.2"
 authors = ["Dr Maxim Orlovsky <orlovsky@ubideco.org>"]
 homepage = "https://strict-types.org"
 repository = "https://github.com/strict-types/strict-encoding"
-rust-version = "1.77.0"
+rust-version = "1.81.0"
```

Comment:
This is a breaking change which can't go into a minor release. To counter-act MSRV bump in dependency just pin it to the latest correct version in Cargo.toml (i.e. add it as a dirrect dependency with "<=" version requirement)

Reply:
Ok, reverted and fixed the dep version